index.html

```diff
@@ -14,6 +14,9 @@
     />
   </head>
   <body>
+    <div class="popup">
```

Comment:
- [ ] Please the popup image should be aligned to the left while the info should be on the right as in the design.
your design 
![image](https://user-images.githubusercontent.com/74744787/226134307-6ca40f37-3db7-4253-b88a-15ce0185fe23.png)
Figma  
![image](https://user-images.githubusercontent.com/74744787/226134323-257bf2b9-eb37-4e35-bc39-c920809ee860.png)